CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+
```

Comment:
@brancz 

This indicates to me that maybe we should add an `[Unreleased]` section to the section and make every PR that needs to be mentioned in the change log also update changelog.md. This has an advantage that this will definitely decrease your burden to write a complete release note once every release. I suggest we give it a try before we automate the changelog gen just like what Kubernetes main repo does.

Reply:
@aramalipoor 

Could you please add a new `[Unreleased]` section before this line just like what [keepchangelog](https://keepachangelog.com/en/1.0.0/#effort) suggests.

Reply:
@andyxning happy to start this practice.

Reply:
My bad for merging this already, do you just want to open a PR to add the title?

Reply:
I will update this in another PR.

Reply:
https://github.com/kubernetes/kube-state-metrics/pull/397